src/table/sticky-scrollbar/styles.scss

```diff
@@ -23,4 +25,28 @@
     // For example it is set as "show when scrolling"
     margin-top: -15px;
   }
+
+  &-offset {
+    // Needed for when sticky columns are enabled.
+    // We offset the scrollbar lower for it not to overlap interactive elements
+    // such as checkboxes or links
+    z-index: 800; // Higher than sticky columns
```

Comment:
What happens if it is not higher but the same than sticky columns?

Reply:
Same problem.

- When no `z-index` was applied (default state without sticky columns enabled), even though the scrollbar is visually on top, it doesn't necessarily intercept click events because it isn't explicitly above the columns in the stacking context. It's in a separate stacking context due to its `position: sticky` property. The columns, being `position: static` (default value) and in a different stacking context, receive the clicks as normal.

- When a `z-index` is applied to both - even if the same value, they are in the same stacking context. The order of elements in the DOM now determines which one is on top. The scrollbar is later in the DOM, so it will be on top and start intercepting the clicks.

Reply:
Thanks for the explanation! So if both have the same `z-index` but `position: sticky` was applied not to the main sticky scrollbar wrapper but to _a child_ of it (and perhaps only this child have non-zero height), so that the scrollbar is again in a different stacking context, would that help?

Reply:
Unfortunately, that also doesn't work with our current sticky scrollbar implementation...